libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1398,17 +1485,18 @@ blk_hash_t PbftManager::identifyLeaderBlock_() {
       continue;
     }
 
-    leader_candidates.emplace_back(std::make_pair(getProposal(v), proposed_block_hash));
+    leader_candidates.emplace_back(std::make_pair(getProposal(v), v));
```

Comment:
why don't just use `getPeriod()` ? this should be more efficient than whole vote object